manifests/init.pp

```diff
@@ -29,6 +31,8 @@
   $slaveof            = undef,
   $version            = 'installed',
   $packages           = ['redis']
+  $redis_conf         = undef,
+  $redis_service      = undef,
```

Comment:
These parameter defaults will yield a non-functional module. Any reason not to move the existing strings to the defaults?